src/hooks-worker-client.coffee

```diff
@@ -163,9 +163,10 @@ class HooksWorkerClient
         callback()
 
   spawnHandler: (callback) ->
-    pathGlobs = [].concat @runner.hooks?.configuration?.options?.hookfiles
+    pathGlobs = [].concat ["--port", "#{@handlerPort}", "--host", "#{@handlerHost}"], @runner.hooks?.configuration?.options?.hookfiles
```

Comment:
Also, lately I tend to prefer to have all Dredd options in the `--option=value` format, consistently. In the case of Dredd itself, both formats should work, I just wanted to choose one way to go and stick with it.